### PR TITLE
Update rnaseq.md

### PR DESCRIPTION
**Pull request by schwenk102**
Formatting table changes. @mccalluc @ngehlenborg

### CODE REVIEW

File: context/app/markdown/docs/assay/rnaseq.md [hunk 1]
```diff
@@ -115,9 +115,9 @@ Figure 6: Plot of per sequence base quality [(Figure from Babraham Bioinformatic
 |  gc_content || FastQC|
 |  sequence_length_distribution |>45 (encode)| FastQC|
 |  sequence_duplication|| FastQC|
-|  k-mer_overrepresentation |20 (accuracy rate 99%)|
+|  k-mer_overrepresentation |20 (accuracy rate 99%)| |
+|  0 |>20 (accuracy rate 99%)| FastQC|
```
Comment (mccalluc) [inline context/app/markdown/docs/assay/rnaseq.md:119]: - Why is this row duplicated?
- "0" doesn't seem like the name of a QC metric?

Comment (mccalluc) [reply context/app/markdown/docs/assay/rnaseq.md:119]: Reference issue number.

Comment (schwenk102) [reply context/app/markdown/docs/assay/rnaseq.md:119]: Originally #726

Comment (schwenk102) [reply context/app/markdown/docs/assay/rnaseq.md:119]: Please delete the second k-mer down through the FastQC and replace with: 

|contamination_of_primers_and_adapters_in_sequencing_data || Library specific data on adapters need to be provided to the read-trimming tool like trimmomatic *(Bioinformatics. 2014 Aug 1; 30(15):2114-20.).*

**Comment by mccalluc** [comment]
@schwenk102 : I believe #764 supersedes this. Closing, but reopen if I'm mistaken.

**Pull request closed without merging**